docs/developer-guide/CLI Methods/generating-the-bundled-config-file.md

```diff
@@ -0,0 +1,87 @@
+# Generating the Bundled Configuration File
+
+[Running Basemaps Locally][running-basemaps-locally] \> [Configuring the Basemaps/server package][configuring-the-basemapsserver-package] \> [Providing a path to a bundled configuration JSON file][path-to-json-config-file] \> Generating the Configuration File
+
+This guide shows you how to generate the bundled configuration file using the **basemaps/cli** package.
+
+## Prerequisites
+
+### AWS Credentials
+
+You'll need **AWS credentials** with permission to read files from the **LINZ AWS S3** bucket at `s3://linz-basemaps`. These credentials will allow you to access and/or generate the bundled configuration file required for running the **Basemaps/server** package.
+
+## Cloning the Repository
+
+### basemaps-config
+
+You must clone the **basemaps-config** repository to your machine.
+
+```bash
+# Clone using the web URL
+git clone https://github.com/linz/basemaps-config.git
+
+# Clone using a password-protected SSH key
```

Comment:
```suggestion
# Clone using a SSH key
```

ssh keys dont have to be password protected, generally you should password protect them